charts/aws-ebs-csi-driver/templates/_node.tpl

```diff
@@ -184,7 +184,17 @@ spec:
           {{- end }}
           {{- with .Values.sidecars.nodeDriverRegistrar.securityContext }}
           securityContext:
-            {{- toYaml . | nindent 12 }}
+            seccompProfile:
+              type: {{ .seccompProfile.type }}
+              {{- if eq .seccompProfile.type "Localhost" }}
+              localhostProfile: {{ .seccompProfile.localhostProfile }}
+              {{- end }}
+            {{- if .readOnlyRootFilesystem }}
+            readOnlyRootFilesystem: {{ .readOnlyRootFilesystem }}
+            {{- end }}
+            {{- if .allowPrivilegeEscalation }}
+            allowPrivilegeEscalation: {{ .allowPrivilegeEscalation }}
+            {{- end }}
```

Comment:
Why are we changing the security context from the existing implementation? This is a breaking change if a user had something like this configured:

```
securityContext:
  runAsUser: 1001
  runAsGroup: 1003
  capabilities:
    add: 'SOME_CAP'
```

(And there's several others missing from the example, like `appArmorProfile` or `procMount`)

I also don't understand why we would even want to be in the business of validating the config here.

Reply:
Good call, based on this feedback I'm inclined to close this PR and keep the defaults as written today. If users wish to configure the `securityContext` beyond the current default values they have the ability to do so.

Reply:
Removed template parsing and left the door open there. Scoped down PR to changing the default seccompProfile config.